app/models/division.rb

```diff
@@ -86,6 +91,7 @@ class Division < ActiveRecord::Base
   has_attached_file :logo, styles: { banner: "840x195>" }
   validates_attachment_content_type :logo, content_type: /\Aimage\/.*\z/
 
+  validate :ascii_encoding
```

Comment:
On second thought, lets just drop this validation. I don't think it accomplishes anything.